shell/platform/darwin/macos/framework/Source/FLEEngine.mm

```diff
@@ -345,6 +345,49 @@ - (void)sendOnChannel:(nonnull NSString*)channel message:(nullable NSData*)messa
   }
 }
 
+- (void)sendOnChannel:(NSString*)channel
+              message:(NSData* _Nullable)message
+          binaryReply:(FlutterBinaryReply _Nullable)callback {
+  FlutterPlatformMessageResponseHandle* response_handle = nullptr;
```

Comment:
Nit: Please define this nearer to where it is used. Right above the `FlutterPlatformMessageCreateResponseHandle` works better.

Reply:
Done